### PR TITLE
Tutorial 1: Making a Simple UI Change

### PR DESCRIPTION
**Pull request by Ash-2k3**
Fix part of #237. 

This PR introduces the first tutorial: Making a Simple UI Change.

### CODE REVIEW

**Comment by Ash-2k3** [comment]
Once the PR is merged, I will update the [Readme](https://github.com/oppia/oppia/edit/develop/.github/README.md) file of Oppia wiki with the link to tutorials.

**Comment by Ash-2k3** [comment]
Hi @seanlip, PTAL.

**Comment by seanlip** [comment]
@Ash-2k3 Could you please resolve the merge conflict?

**Comment by Ash-2k3** [comment]
Done @seanlip.

**Review by seanlip — CHANGES_REQUESTED** [review]
@Ash-2k3 Looks good, can you please look into https://stackoverflow.com/questions/50544499/how-to-make-a-styled-markdown-admonition-box-in-a-github-gist for making the questions "pop out" more?

Also do we need to actually include this in the README? @hrshkshri recently added a section in the sidebar for getting started, I figured we could add Tutorials there as well. (I'd suggest putting it after the Populating data on the local server" bullet point -- have a "Tutorials" link and then sub-bullet points for each tutorial.) Could you please do this in this PR?

Thanks!

**Comment by Ash-2k3** [comment]
> @Ash-2k3 Looks good, can you please look into https://stackoverflow.com/questions/50544499/how-to-make-a-styled-markdown-admonition-box-in-a-github-gist for making the questions "pop out" more?
> 
> Also do we need to actually include this in the README? @hrshkshri recently added a section in the sidebar for getting started, I figured we could add Tutorials there as well. (I'd suggest putting it after the Populating data on the local server" bullet point -- have a "Tutorials" link and then sub-bullet points for each tutorial.) Could you please do this in this PR?
> 
> Thanks!

Done, I have addressed it, PTAL.

File: Tutorial-1.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+**Setup:**
+Start your local Oppia server. If you haven’t installed Oppia on your local machine, please follow the steps in [this wiki page](https://github.com/oppia/oppia/wiki/Installing-Oppia).
+
+**Identify the Element:**
+When the server is started, it navigates to the home page by default. Since the changes have to be done in the About page, let’s head to the About page at localhost:8181/about.
+
+<img src="images/tutorial-1/HomePage.png" alt="Practice 1 Image">
+
+> [!NOTE]
```
Comment (Ash-2k3) [inline Tutorial-1.md]: Hi @seanlip FYI, for this line when I review changes in file tab I get something like this  
Screenshot 2024-01-21 at 20 23 33
 But when i go to edit mode in file tab I see something like this in preview

Screenshot 2024-01-21 at 20 24 11

Comment (seanlip) [reply Tutorial-1.md]: I think it's probably fine, my only concern is "Note" which indicates something skippable. Are there any other options we can use that are more along the lines of "Do this" or "Attention"?

File: _Sidebar.md [hunk 2]
```diff
@@ -14,10 +14,10 @@
 **Developing Oppia**
   * [[FAQs|Frequently-Asked-Questions]]
   * [[Installing Oppia|https://github.com/oppia/oppia/wiki/Installing-Oppia]]
-  * Tutorials
-    * [[Tutorial 1|Tutorial-1]]
   * Getting started with the codebase
     * [[Populating data on local server|Populating-data-on-local-server]]
+    * Tutorials
+      * [[Tutorial 1|Tutorial-1]]
```
Comment (seanlip) [inline _Sidebar.md]: No one knows what "tutorial 1" is -- give it a name like "Make a simple UI change".

**Review by seanlip — CHANGES_REQUESTED** [review]

File: _Sidebar.md (hunk 2 above)
Comment (Ash-2k3) [reply _Sidebar.md]: Done, Thanks.

File: Tutorial-1.md (hunk 1 above)
Comment (Ash-2k3) [reply Tutorial-1.md]: Okay, I saw one labelled as 'Attention' in documentation so I have replaced Note with it. Thanks.

File: _Sidebar.md [hunk 3]
```diff
@@ -17,7 +17,7 @@
   * Getting started with the codebase
     * [[Populating data on local server|Populating-data-on-local-server]]
     * Tutorials
-      * [[Tutorial 1|Tutorial-1]]
+      * [[Make a Simple UI change|Tutorial-1]]
```
Comment (seanlip) [inline _Sidebar.md]: Why capitalize "Simple" but not "change"?

Suggest just keeping both of those lowercase.

Comment (Ash-2k3) [reply _Sidebar.md]: Done.

**Comment by seanlip** [comment]
Also @Ash-2k3 I'm not toally sure about IMPORTANT, but our options might be limited (though you can perhaps try the emoji approach here too: https://stackoverflow.com/a/57118885. I suggest we just launch this and perhaps you can get feedback from folks on it, then make follow-up changes as needed.

**Review by seanlip — APPROVED** [review]

**Comment by seanlip** [comment]
Also @Ash-2k3 congrats on your (and Oppia's) first tutorial! :tada:

**Comment by Ash-2k3** [comment]
Thanks @seanlip 🥇

**Pull request merged**